edx_solutions_api_integration/users/views.py

```diff
@@ -732,6 +734,21 @@ def post(self, request, user_id):  # pylint: disable=R0915
         existing_user.save()
 
         username = request.data.get('username', None)
+        old_username = str(existing_user.username)
```

Comment:
@qazi-fahid this block should run only when username gets updated, please see [L765](https://github.com/edx-solutions/api-integration/pull/214/files#diff-6e10f5856db69f12fe0c2aef7e4ec7dfR765)

Reply:
done now the code runs after the username gets updated and also done the version bump